ArtOfIllusion/src/artofillusion/MeshEditorWindow.java

```diff
@@ -1484,11 +1484,12 @@ public boolean canAcceptAsParent(TreeElement el)
         return false;
       }
     }
-    for (int i = 0; i < theScene.getNumObjects(); i++)
+    for (ObjectInfo info: theScene.getObjects())
     {
-      ObjectInfo info = theScene.getObject(i);
       if (info.getParent() == null)
+      {
```

Comment:
Does your IDE prompt you to add these braces? They are not necessary. The Single-Line version is an established part of AOI's code style.